src/audio/volume/Kconfig.hifi

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+comment "volume optimization level select"
+
+choice "VOLUME_HIFI_LEVEL_SELECT"
+	prompt "choose which hifi level used for volume module"
+
+	config VOLUME_DEFAULT_OPT_LEVEL
+		prompt "dummy optimized option for volume module"
+		bool
+		help
+			This option is dummy option for volume module, when this was
```

Comment:
It's not dummy, it just generic C using standard arch ISA

Reply:
So not dummy, but not generic either, see above.